### ISSUE TITLE
Remove all panics from MVBA

### ISSUE THREAD

**Issue by davidrusu** [maidsafe/sn_consensus#91]
We have unwraps scattered throughout the MVBA code, this was done to aid prototyping but now we need to prepare the code for use.

We need to rewrite those parts so that either the panicking code path is not representable (i.e. by refactoring the design/structs/enums) or by returning errors when we can't find a good refactor.

Current count of unwraps by file:

```
$ rg -c 'unwrap\(' | rg 'mvba' | rg -v 'tests'

src/mvba/abba/mod.rs:2
src/mvba/vcbc/mod.rs:2
src/mvba/consensus.rs:12
src/mvba/mvba/mod.rs:4
src/mvba/broadcaster.rs:2
```
Total: 22

**Comment by davidrusu** [comment]
Updated counts with #92 in

```
$ rg -c 'unwrap\(' | rg 'mvba' | rg -v 'tests'                          
src/mvba/abba/mod.rs:2
src/mvba/broadcaster.rs:2
src/mvba/consensus.rs:12
src/mvba/mvba/mod.rs:4
```

Total: 20

**Comment by davidrusu** [comment]
fixed by #102
